CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Fix: explicitAttributes true case: should progress just active attributes which receives measures and all active attributes with expressions

```

Comment:
This behaviour match the one described in documentation?

Reply:
According to this:

https://github.com/telefonicaid/iotagent-node-lib/blob/ea48c800eaca591441a70be56b34d61b93b93dc9/doc/advanced-topics.md?plain=1#L301-L309

> *the IoTAgent rejects the  measure elements that are not defined in the mappings of device or group provision*

From my perspective, yes, it impact on it

Reply:
https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/advanced-topics.md#explicitly-defined-attributes-explicitattrs

Case 2:

"explicitAttrs": true

just measures defined in active, static (plus conditionally TimeInstant) will be propagated to NGSI interface.

Reply:
So, no further changes required in the documentation

Reply:
NTC